packages/cli/src/fs.ts

```diff
@@ -17,5 +18,10 @@ export const fsFromUrl = async (url: string, localBaseDir: string) => {
     // eslint-disable-next-line @typescript-eslint/return-await
     return await googleCloudStorageFs(url)
   }
+  if (fsType === 'az') {
```

Comment:
Thanks for the contribution!

The type of the fsType as well as the URL should use `azblob` protocol and be conforming to this format:
https://gocloud.dev/howto/blob/#azure

#102

Reply:
I saw this link in the issue but I got confused as all of the Azure documentation was using `https://`. I guess I'll have to understand the whole architecture of Preevy deeply in order to figure this out.